Dockerfile

```diff
@@ -1,37 +1,67 @@
-# To run: docker run -v /path/to/wsgi.py:/var/www/sheepdog/wsgi.py --name=sheepdog -p 81:80 sheepdog
-# To check running container: docker exec -it sheepdog /bin/bash
+# To run:
+# - Create and fill out `creds.json`:
+# {
+#   "fence_host": "",
```

Comment:
I don't remember if I hacked the config in helm, but can we also read these from environment variables?